Update Notes:

Bokaiii opened the pull request:
- Production fxml
- ProductionController dynamic list of credits

Bokaiii committed:
Update Notes:
- Production fxml
- ProductionController dynamic list of credits

nymann commented:
Ideally we make unit tests as well for this, but start with fixing the 5 code smells: https://sonarcloud.io/project/issues?id=creditoro_desktop-client&pullRequest=68&resolved=false&types=CODE_SMELL 
@Bokaiii

Bokaiii committed:
Update Notes:
- Added Production as Views ENUM and FXML map
- ProductionController get list from ViewModel
- ViewModel temp list

Joergenson committed:
Update Production.java

Bokaiii committed:
Merge branches 'master' and 'production' of https://github.com/creditoro/desktop-client into production

Joergenson committed:
Update ProductionTest.java

Joergenson committed:
Update ProductionController.java

Joergenson committed:
Update ProductionController.java

Joergenson commented on src/main/java/dk/creditoro/client/networking/rest_client/endpoints/ChannelsEndpoint.java:
```diff
@@ -28,12 +29,23 @@ public Channel patchChannel(String identifier, Map<String, Object> fields) {
         return response.asObject(Channel[].class).getBody();
     }
 
+	// This is never gonna work as it is setup now? - kevin 06-05-2020
```
What do you mean?

Bokaiii committed:
Merge branch 'master' of https://github.com/creditoro/desktop-client into production

 Conflicts:
	src/main/java/dk/creditoro/client/core/ViewHandler.java
	src/main/java/dk/creditoro/client/core/Views.java
	src/main/java/dk/creditoro/client/model/crud/Production.java
	src/test/java/dk/creditoro/client/model/crud/ProductionTest.java

Bokaiii committed:
Update Notes:
- Production ViewModel
- Created credit class

Bokaiii committed:
Update Notes:
- Fixed Code smells
- Added Search Credits Functionality
- Added ICreditModel
- Added CreditModel
- Added Person class
- Updated production FXML
- Updated production Controller

Bokaiii committed:
Update Notes:
- Fixed Code smell

Joergenson committed:
Update RestClient.java

Joergenson committed:
Update RestClient.java
